cfme/tests/containers/test_manageiq_ansible_users.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+from cfme.containers.provider import ContainersProvider
+from cfme.configure.access_control import User
+from cfme.web_ui import Table
```

Comment:
**REQUIRED**
Whole this test must be rewritten because of this. New code should use `widgetastic` framework.